dir.props

```diff
@@ -124,6 +124,27 @@
   </PropertyGroup>
 
   <Import Project="$(BuildConfigurationImportFile)" Condition="Exists('$(BuildConfigurationImportFile)')" />
+  <PropertyGroup>
+    <_runtimeOSVersionIndex>$(RuntimeOS.IndexOfAny(".-0123456789"))</_runtimeOSVersionIndex>
+    <_runtimeOSFamily Condition="'$(_runtimeOSVersionIndex)' != '-1'">$(RuntimeOS.SubString(0, $(_runtimeOSVersionIndex)))</_runtimeOSFamily>
+    <_portableRID>linux</_portableRID>
+    <_portableRID Condition="'$(_runtimeOSFamily)' == 'win'">win</_portableRID>
+    <_portableRID Condition="'$(_runtimeOSFamily)' == 'osx'">osx</_portableRID>
+    <!-- support cross-targeting by choosing a RID to restore when running on a different machine that what we're build for -->
+    <_portableRID Condition="'$(OSGroup)' == 'Unix' AND '$(_runtimeOSFamily)' != 'osx'">linux</_portableRID>
+    <_portableRID Condition="'$(OSGroup)' == 'Windows_NT'">win</_portableRID>
+
+    <_packageRID/>
+    <_packageRID Condition="'$(PortableBuild)' == 'true'">$(_portableRID)-$(ArchGroup)</_packageRID>
+    <_packageRID Condition="'$(TargetGroup)' == 'uap'">win10-$(ArchGroup)</_packageRID>
+    <_packageRID Condition="'$(TargetGroup)' == 'uapaot'">win10-$(ArchGroup)-aot</_packageRID>
+    <PackageRID Condition="'$(PackageRID)' == ''">$(_packageRID)</PackageRID>
+    <PackageRID Condition="'$(PackageRID)' == ''">$(RuntimeOS)-$(ArchGroup)</PackageRID>
+
+    <_runtimeOS>$(RuntimeOS)</_runtimeOS>
+    <_runtimeOS Condition="'$(_runtimeOS)' == 'tizen.4.0.0'">ubuntu.14.04</_runtimeOS>
+    <RuntimeRID>$(_runtimeOS)-x64</RuntimeRID>
```

Comment:
> We should switch to using the portable RID here

Yes, when doing a portable build. When a maintainer does a distro specific build, he may want to use his own runtime to achieve that build.

> think about whether or not we want to hard code it to x64

This value should be provided by BuildTools then (like RuntimeOS).

> perhaps we should change this to be ToolRuntimeRID

I also think RuntimeRID doesn't convey the meaning well. To properly rename it, the change should also be made to the Buildtools and corefx config, since those generate/accept RuntimeOS.
To suggest another alternative: HostRID. (Buildtools could provide HostOS and HostArch.)